compiler/rustc_hir_typeck/src/coercion.rs

```diff
@@ -850,6 +850,8 @@ impl<'f, 'tcx> Coerce<'f, 'tcx> {
                     }
 
                     // Safe `#[target_feature]` functions are not assignable to safe fn pointers (RFC 2396).
+                    // FIXME: should this be true also for functions that inherit target features
```

Comment:
This FIXME should probably have been marked with the feature name, to make it clear to search for these FIXMEs when later determining what this feature's status is.

Also what do you mean by witness types? Do you mean arguments that have struct features? It's probably best to avoid calling them witnesses given that that already has a meaning for coroutines.